packages/cactus-cmd-api-server/src/main/typescript/api-server.ts

```diff
@@ -109,8 +109,10 @@ export class ApiServer {
       throw new Error(`ApiServer#ctor options.config was falsy`);
     }
 
-    this.enableShutdownHook = Bools.isBooleanStrict(options.enableShutdownHook)
-      ? (options.enableShutdownHook as boolean)
+    this.enableShutdownHook = Bools.isBooleanStrict(
```

Comment:
@m-courtin Could you also add a unit test written in Jest (not tap/tape-promise) that asserts for both the `true` and `false` cases that after the instantiation of the API server, the config value ends up as one would expect it to?

Reply:
@petermetz Yes, sure thing. I will add a corresponding UT

Reply:
@m-courtin Great, thank you very much!